Data/models/Bookmark+Sync.swift

```diff
@@ -15,14 +15,48 @@ extension Bookmark {
     /// Base order is needed to distinguish between bookmarks on different devices and platforms.
     static var baseOrder: String { return Preferences.Sync.baseSyncOrder.value }
     
-    /// Sets order for all bookmarks. Needed after user joins sync group for the first time.
+    // Errors thrown when using the javascript library for calculating sync orders are returned
+    // as 'undefined'.
+    private var isSyncOrderValid: Bool { return syncOrder != nil && syncOrder != "undefined" }
```

Comment:
Is this something we can't fix on the JS end?

Reply:
I will make a validator so js stuff like `null` `undefined` won't be returned